KmsClient.js

```diff
@@ -10,13 +10,21 @@ import {signCapabilityInvocation} from 'http-signature-zcap-invoke';
 const SECURITY_CONTEXT_V2_URL = 'https://w3id.org/security/v2';
 const DEFAULT_HEADERS = {Accept: 'application/ld+json, application/json'};
 
+/**
+ * A node https module.
+ *
+ * @typedef {object} https
+ * @see https://nodejs.org/docs/latest-v11.x/api/https.html
```

Comment:
I don't understand this... is this a copy+paste error?

Reply:
Remove it here: https://github.com/digitalbazaar/web-kms-client/pull/2/commits/09d4ec93574132f2a7a5205559256f63a81ca903